src/parser/apiSpec.yaml

```diff
@@ -0,0 +1,126 @@
+swagger: "2.0"
+info:
+  description: "API description for the Parser/Database component of the Road Search Engine."
+  version: "1.0.0"
+  title: "Parser/Database Component"
+  contact:
+    email: "jaimehisao@google.com"
+host: "localhost"
+basePath: "/v1"
+schemes:
+- "https"
+- "http"
+paths:
+  /parse:
+    post:
+      tags:
+      - "parser"
```

Comment:
This doesn't match any tag. I believe it should be something like:
tags:
-name: "parser"
  description: "your description"

Reply:
the `tag` is the name of the section where the `/parse` lives